frontend/src/Game.js

```diff
@@ -27,9 +28,9 @@ export default class Game {
         this.gold = 250;
     }
 
-    setSteps = (spawn, goal) => {
-        const path = djikstra(this.gameMap, spawn, goal);
```

Comment:
What does `djikstra` do with this? Should it have the same default as `getSteps()`?

Reply:
`djikstra` gets the path from tile to tile, e..g `[0,0,0],[1,0,0]` 
`setSteps breaks the path down into small increments, eg `[0,0,0],[0.02,0,0],[0.04,0,0]...`

Reply:
Opps, put this comment on the wrong side. I was referring to passing `speed` to `djikstra`. Since it's just giving you the path, doesn't seem like it's needed in the function call then.

Reply:
Good catch, djikstra doesn't care about speed.